src/popup/components/Modals/ConfirmTransactionSign.vue

```diff
@@ -244,10 +245,11 @@ export default {
         setTimeout(() => { this.loading = false; }, 20000);
         const sdk = await getSdk();
         const { bytecode } = await sdk.getContractByteCode(this.transaction.contractId);
-        const txParams = await sdk.compilerApi.decodeCalldataBytecode({
-          bytecode,
-          calldata: this.transaction.callData,
-        });
+        const txParams = await new CompilerHttp(this.activeNetwork.compilerUrl)
```

Comment:
Not initializing the compilerHttp at app bootup but gets initialized every time while signing a transaction. Hence, the app loads faster.  It's not often we make multiple transaction signings at once.

Reply:
```suggestion
        const txParams = new CompilerHttp(this.activeNetwork.compilerUrl)
```
Compiler initialises in sync the same as node, so shouldn't be a problem to create an instance somewhere global.

Reply:
Okay.

Reply:
Part of the sdk now.